src/root/components/highlighted-operations/operation-card.js

```diff
@@ -14,13 +14,18 @@ const OperationCard = ({operationId, operationName, emergencyDeployments, appeal
   return (
     <div className='key-emergencies-item' key={operationId}>
       <Link to={`/emergencies/${operationId}`}>
-        <div className="card_box card_box_left">
-          <h2 className='card__title'>{ operationName.length > 30 ? operationName.slice(0, 30) + '...' : operationName }</h2>
-          <small className='last_updated'>
-            <Translate stringId='operationCardLastUpdated'/>
-            &nbsp;
-            {formatDate(lastUpdate)}
-          </small>
+        <div className="card_box card_box_left row">
+          <div className='card__title__wrap'>
+            <h2 className='card__title'>{ operationName.length > 30 ? operationName.slice(0, 30) + '...' : operationName }</h2>
```

Comment:
Not that these are change in this PR but just wanted to point out one thing here:
We should use CSS to properly truncate the text, 
```css
.card__title {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}
```
for more info: https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
Okay. Is that a requirement that the title is only one line? Then we can do the needful @frozenhelium 
I suppose we can also make a mixin and reuse as needed...

Reply:
@imohkay right now the javascript is doing the truncation to 30 characters in the `.slice(...)` operation - I think the idea here would be to not truncate in Javascript and have CSS handle the display with ellipsis for overflow. Happy to help with the JS end of this if you can implement the text-overflow: ellipsis bit to look neat, etc.

Reply:
Per voice with @imohkay - let's ticket and do this separately.